source/unified-test-format/unified-test-format.rst

```diff
@@ -1006,7 +1045,7 @@ has been created.
 
 This operation proxies the client's ``watch`` method and supports the same
 arguments and options. Test files SHOULD NOT use the client's ``watch``
-operation directly for reasons discussed in `ChangeStream Operations`_. Test
+operation directly for reasons discussed in `Supported Entity Types`_. Test
```

Comment:
Consider creating an anchor and linking to the change stream entity type directly (same for the following two links).

Reply:
Done.